index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Harshika's Portfolio</title>
+    <link rel="stylesheet" href="style.css" class="css">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@800&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://kit.fontawesome.com/3ab8daaa6e.css" crossorigin="anonymous">
+    <script src="https://kit.fontawesome.com/3ab8daaa6e.js" crossorigin="anonymous"defer></script>
+</head>
+<body>
+    <div class="mobileinfo">
+        <h1>Welcome</h1>
+          </div>
+          <div class="mobilemenu">
+          <i class="fa-sharp fa-solid fa-bars"></i>  </div>
+          
+    <div class="container header">
+        <div class="flex-menu">
+            <a href="#">Portfolio</a>
+        </div>
+        <div class="flex-menu">
+            <a href="#">About</a>
+        </div>
+        <div class="flex-menu">
+            <a href="#">Contact</a>
+        </div>
+
+    </div>
+    <div class="container main">
```

Comment:
- [ ] Kindly ensure that the background images don't appear to be colliding with the elements of your page.
 
![image](https://user-images.githubusercontent.com/98453979/219411704-84a18387-9258-419b-80c9-8fa543c357b0.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/219411844-679dec8f-d164-4c27-ab76-8ab344ebda67.png)